biodatacatalyst-ui/src/main/webapp/picsureui/studyAccess/studyAccess.hbs

```diff
@@ -0,0 +1,313 @@
+<style>
```

Comment:
These styles could also go in overrides/styles.css, unless they conflict with existing styles in there.

Reply:
moved as suggested